introduction.md

```diff
@@ -1 +1,6 @@
-Let's try listing my favorite path as JavaScript and see if that works.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?